certbot/client.py

```diff
@@ -317,7 +317,13 @@ def save_certificate(self, certr, chain_cert,
 
         cert_pem = OpenSSL.crypto.dump_certificate(
             OpenSSL.crypto.FILETYPE_PEM, certr.body.wrapped)
-        cert_file, act_cert_path = le_util.unique_file(cert_path, 0o644)
+
+        if cert_path != constants.CLI_DEFAULTS['auth_cert_path']:
```

Comment:
You have the option of doing `cli.set_by_cli('cert_path')`, which will return True if the user explicitly set `cert_path`, even if they set it to the default.  Up to you about whether you believe that is the more correct behaviour.